src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal static class ConfigPathsRunner
```

Comment:
Can this runner be moved to `NuGet.Commands` project? https://github.com/NuGet/NuGet.Client/tree/dev/src/NuGet.Core/NuGet.Commands

Reply:
I had provided the exact opposite feedback earlier. https://github.com/NuGet/NuGet.Client/pull/5088#discussion_r1137520319
This implementation will only be a part of the .NET CLI, so there's no point in tying it to a public API unnecessarily.

Reply:
Make sense Nikolche. I am resolving this thread.